tools/multibuild/Dockerfile.package

```diff
@@ -0,0 +1,60 @@
+# Dockerfile.package
+# A dockerfile for packaging SSH No Ports releases using docker buildx
+
+FROM atsigncompany/buildimage:3.5.2_3.6.0-149.3.beta@sha256:df67b9e3271381fc0c5b20e7350cf4de8dad6ac62e075b49b1a866c49af47409 AS build
+# Using atsigncompany/buildimage until official dart image has RISC-V support
+WORKDIR /noports
+
+# install node for later (keep at the top file to increase cache hits)
+# hadolint ignore=DL3008
+RUN apt-get update; \
+  apt-get install -y --no-install-recommends npm
+
+COPY . .
+
+# Build packages/dart/sshnoports
+WORKDIR /noports/packages/dart/sshnoports
+RUN set -eux; \
+  mkdir -p /sshnp/debug; \
+  mkdir -p /sshnp/beta/web/admin; \
+  mkdir /tarball; \
+  dart pub get --enforce-lockfile; \
+  dart run build_runner build --delete-conflicting-outputs; \
+  dart compile exe bin/activate_cli.dart -v -o /sshnp/at_activate; \
+  dart compile exe bin/sshnp.dart -v -o /sshnp/sshnp; \
+  dart compile exe bin/npt.dart -v -o /sshnp/npt; \
+  dart compile exe bin/npp_file.dart -v -o /sshnp/npp_file; \
+  ln -s sshnp/npp_file sshnp/npa_file; \
```

Comment:
Multibuild did not like this:
`https://github.com/atsign-foundation/noports/actions/runs/10721898228/job/29731556532`

I think it is okay to make breaking changes in the alpha release and just remove npa_file and leave in npp_file

Reply:
I see why, needs to be absolute path, but I still don't think a symbolic link in the release is a good idea